R/gmailr.R

```diff
@@ -38,61 +42,96 @@ get_token <- function() {
 #' Clear the current oauth token
 #' @export
 clear_token <- function() {
```

Comment:
Maybe we should just remove this function entirely? Does gargle have an equivalent?

Reply:
ATM gargle does not because holding the token in memory is currently entirely the job of the client package. It's still not clear whether and how to edit the cache file. This needs to get sorted out.

Reply:
For example, it should be easier to remove tokens selectively vs. deleting the entire cache.